src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -127,7 +127,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <When Condition="'$(RuntimeFrameworkName)' == 'Microsoft.AspNetCore.All'">
 
       <PropertyGroup>
-        <DefaultPatchVersionForAspNetCoreAll2_1 Condition="'$(DefaultPatchVersionForAspNetCoreAll2_1)' == ''">2.1.1</DefaultPatchVersionForAspNetCoreAll2_1>
+        <DefaultPatchVersionForAspNetCoreAll2_1 Condition="'$(DefaultPatchVersionForAspNetCoreAll2_1)' == ''">2.1.3</DefaultPatchVersionForAspNetCoreAll2_1>
```

Comment:
Isn't this supposed to be 2.1.1?

Reply:
@natemcmaster is this in anticipation for https://github.com/aspnet/Universe/issues/1256?